blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -188,38 +191,43 @@ const Results = ({
 	const onReadMoreClick = useCallback(() => {
 		setQueryOffset((oldOffset) => oldOffset + configuredSize);
 	}, [configuredSize, setQueryOffset]);
-
 	return viewableElements?.length > 0 && !isServerSideLazy ? (
-		<div className="results-list-container">
+		<Stack>
 			{viewableElements.map((element, index) => (
-				<ResultItem
+				<Stack
 					key={`result-card-${element._id}`}
-					ref={elementRefs[index]}
-					arcSite={arcSite}
-					element={element}
-					imageProperties={imageProperties}
-					placeholderResizedImageOptions={placeholderResizedImageOptions}
-					showByline={showByline}
-					showDate={showDate}
-					showDescription={showDescription}
-					showHeadline={showHeadline}
-					showImage={showImage}
-					showItemOverline={showItemOverline}
-					targetFallbackImage={targetFallbackImage}
-				/>
+					gap="1rem"
+					className={`${BLOCK_CLASS_NAME}__${index === 0 ? "container--first" : "container"}`}
+				>
+					<ResultItem
+						ref={elementRefs[index]}
+						arcSite={arcSite}
+						element={element}
+						imageProperties={imageProperties}
+						placeholderResizedImageOptions={placeholderResizedImageOptions}
+						showByline={showByline}
+						showDate={showDate}
+						showDescription={showDescription}
+						showHeadline={showHeadline}
+						showImage={showImage}
+						showItemOverline={showItemOverline}
+						targetFallbackImage={targetFallbackImage}
+					/>
+					<Divider />
+				</Stack>
 			))}
 			{isThereMore && (
-				<div className="see-more">
+				<Stack alignment="center" className={`${BLOCK_CLASS_NAME}__see-more`}>
 					<Button
-						ariaLabel={phrases.t("results-list-block.see-more-button-aria-label")}
-						buttonStyle={BUTTON_STYLES.PRIMARY}
-						buttonTypes={BUTTON_TYPES.LABEL_ONLY}
+						accessibilityLabel={phrases.t("global.see-more-button-aria-label")}
+						variant="primary"
 						onClick={onReadMoreClick}
-						text={phrases.t("results-list-block.see-more-button")}
-					/>
-				</div>
+					>
+						{phrases.t("global.see-more-button")}
+					</Button>
+				</Stack>
 			)}
-		</div>
+		</Stack>
```

Comment:
More detail in suggested changes here...

Reply:
You will want to put the `key={`result-card-${element._id}`}` back on the ResultItem in the suggestion...

Reply:
![image](https://user-images.githubusercontent.com/2287238/200930883-0d6138b8-f5f9-4d6a-b292-d3864cf18246.png)